### PR TITLE
feat(opentrons-ai-client): add ChatDisplay component

### PR DESCRIPTION
**Pull request by koji**
# Overview
Add ChatDisplay component, its test, and its stories.

The display in Storybook isn't the same as the design since the component doesn't have markdown parse function yet.
The function will be implemented in a following PR.

close AUTH-209 partially

# Test Plan

# Changelog
- add ChatDisplay component, its test, and its stories.

# Review requests
If you have any better idea for the component name, please let me know. (I don't like the current component name...)

# Risk assessment
low

### CODE REVIEW

File: opentrons-ai-client/src/molecules/ChatDisplay/__tests__/ChatDisplay.test.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    props = {
+      text: 'mock text from the backend',
+      isUserInput: false,
+    }
+  })
+  it('should display response from the backend and label', () => {
+    render(props)
+    screen.getByText('OpentronsAI')
+    screen.getByText('mock text from the backend')
+    // Note (kk:04/16/2024) activate the following when jsdom's issue is solved
```
Comment (jerader) [inline opentrons-ai-client/src/molecules/ChatDisplay/__tests__/ChatDisplay.test.tsx]: not sure if marking these comments as TODOs would be better?

File: opentrons-ai-client/src/molecules/ChatDisplay/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  isUserInput,
+}: ChatDisplayProps): JSX.Element {
+  const { t } = useTranslation('protocol_generator')
+  return (
+    <Flex
+      flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing12}
+      paddingLeft={isUserInput ? SPACING.spacing40 : undefined}
+      paddingRight={isUserInput ? undefined : SPACING.spacing40}
+      //   max-width="58.125rem"
```
Comment (jerader) [inline opentrons-ai-client/src/molecules/ChatDisplay/index.tsx:28]: will you add this back later?

File: opentrons-ai-client/src/molecules/ChatDisplay/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+interface ChatDisplayProps {
+  text: string
+  isUserInput: boolean
```
Comment (jerader) [inline opentrons-ai-client/src/molecules/ChatDisplay/index.tsx:14]: this boolean prop seems okay for now. Is there ever a chance we will allow multi users in the chat at once? If so, maybe changing this to a prop called `input` and then `InputType: 'backend' | 'user'` could be better. Feel free to change the prop/type name!

**Review by jerader — APPROVED** [review]
lgtm, i left a few suggestions!

File: opentrons-ai-client/src/molecules/ChatDisplay/index.tsx (hunk 2 above)
Comment (koji) [reply opentrons-ai-client/src/molecules/ChatDisplay/index.tsx:28]: Still waiting for the answer from Sue

File: opentrons-ai-client/src/molecules/ChatDisplay/index.tsx (hunk 3 above)
Comment (koji) [reply opentrons-ai-client/src/molecules/ChatDisplay/index.tsx:14]: I don't think we will allow multi users to use the chat at once.
Actually, I still thinking of the implementation of user input part.

**Pull request merged**